docs/README.md

```
@@ -0,0 +1,22 @@
+# AIQLeads Documentation
+
+## Overview
+This repository contains all documentation for the AIQLeads platform.
+
+## Structure
+- `/core` - Core project documentation
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)